rocketpy/simulation/monte_carlo.py

```diff
@@ -0,0 +1,726 @@
+        dictionary.
+
+        Parameters
+        ----------
+        filename : str
+            Name or directory path to the file to be imported. If none,
+            self.filename will be used.
+
+        Returns
+        -------
+        None
+        """
+        # select file to use
+        filepath = filename if filename else self.filename
+
+        try:
+            with open(f"{filepath}.errors.txt", "r+", encoding="utf-8"):
+                self.error_file = f"{filepath}.errors.txt"
+                # Print the number of flights simulated
+                print(f"The following error file was imported: {filepath}.errors.txt\n")
+        except FileNotFoundError:
+            with open(filepath, "r+", encoding="utf-8"):
+                self.error_file = filepath
+                # Print the number of flights simulated
+                print(f"The following error file was imported: {filepath}\n")
+
+    def import_results(self, filename=None):
+        """Import monte carlo results from .txt file and save it into a
+        dictionary.
+
+        Parameters
+        ----------
+        filename : str
+            Name or directory path to the file to be imported. If none,
+            self.filename will be used.
+
+        Returns
+        -------
+        None
+        """
+        # select file to use
+        filepath = filename if filename else self.filename
+
+        self.import_outputs(filename=filepath)
+        self.import_inputs(filename=filepath)
+        self.import_errors(filename=filepath)
+
+    def export_ellipses_to_kml(
+        self,
+        filename,
+        origin_lat,
+        origin_lon,
+        type="all",
+        resolution=100,
+        color="ff0000ff",
+    ):
+        """Generates a KML file with the ellipses on the impact point.
+
+        Parameters
+        ----------
+        results : dict
+            Contains results from the Monte Carlo simulation.
+        filename : String
+            Name to the KML exported file.
+        origin_lat : float
+            Latitude coordinate of Ellipses' geometric center, in degrees.
+        origin_lon : float
+            Latitude coordinate of Ellipses' geometric center, in degrees.
+        type : String
+            Type of ellipses to be exported. Options are: 'all', 'impact' and
+            'apogee'. Default is 'all', it exports both apogee and impact
+            ellipses.
+        resolution : int
+            Number of points to be used to draw the ellipse. Default is 100.
+        color : String
+            Color of the ellipse. Default is 'ff0000ff', which is red.
+            Kml files use an 8 digit HEX color format, see its docs.
+
+        Returns
+        -------
+        None
+        """
+
+        (
+            impact_ellipses,
+            apogee_ellipses,
+            _,
+            _,
+            _,
+            _,
+        ) = generate_monte_carlo_ellipses(self.results)
```

Comment:
```suggestion
        (
            impact_ellipses,
            apogee_ellipses,
            *_,
        ) = generate_monte_carlo_ellipses(self.results)
```

Reply:
Thanks for the suggestion, fixed here: [0d783df](https://github.com/RocketPy-Team/RocketPy/pull/232/commits/0d783df723c99678bfb3ede186b43d655a631a36)